paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetConfigurationKtx.kt

```diff
@@ -28,11 +28,26 @@ internal fun PaymentSheet.Configuration.validate() {
                     " the Customer ID cannot be an empty string."
             )
         }
-        customer?.ephemeralKeySecret?.isBlank() == true -> {
-            throw InvalidParameterException(
-                "When a CustomerConfiguration is passed to PaymentSheet, " +
-                    "the ephemeralKeySecret cannot be an empty string."
-            )
+    }
+
+    customer?.accessType?.let { customerAccessType ->
```

Comment:
Should we also test the prefixes for these?

Reply:
Figuring out with John whether this is potentially risky to do client-side if there is a format change in the ephemeral key secret or customer session client secret pattern. I'll make a follow-up PR for prefixes.